LinhLT - update cảnh báo prometheus

linhlt247 opened the pull request:
- xử lý các vấn đề tồn tại trong bài thuyết trình

linhlt247 committed:
update alert manager

linhlt247 committed:
update

linhlt247 committed:
update cảnh báo

linhlt247 committed:
update

ducnc commented on LinhLT/Prometheus+grafana/README.md:
```diff
@@ -118,6 +119,77 @@ Long-lived jobs/Exporter: Là những job sẽ tồn tại lâu dài. Các Expor
+    - Slave IO running.
+    - Slave SQL running.
+    - Seconds behind master.
+- Các bạn xem tại đây: https://github.com/linhlt247/networking-team/tree/master/LinhLT/Prometheus%2Bgrafana/mysql%20exporter%20python
```
Chỉnh lại link về hocchudong nhé

linhlt247 replied:
Done

ducnc commented on LinhLT/Prometheus+grafana/README.md:
```diff
@@ -159,10 +231,39 @@ cần monitor.
 
 - Sau khi cấu hình xong, các bạn có thể vào địa chỉ: `http://ip:9090/targets` để kiểm tra
 
+- Ví dụ đơn giản để push metrics:
```
Thống nhất tài liệu khi viết về `http://ip:9091` hoặc `http://pushgateway.example.org` để tiện theo dõi, dễ hình dung đâu là push gateway, đâu là prometheus server

linhlt247 replied:
Done

ducnc commented on LinhLT/Prometheus+grafana/README.md:
````diff
@@ -159,10 +231,39 @@ cần monitor.
+```sh
+echo "some_metric 3.14" | curl --data-binary @- http://ip:9091/metrics/job/some_job
+```
+- Push metrics với nhiều thông tin hơn:
````
Em mô tả rõ thêm cách thực hiện dưới Short-live như thế nào, Push GW như thế nào, Prometheus như nào

linhlt247 replied:
Done

linhlt247 committed:
update

update link.
bổ sung thêm các thông tin về pushgateway, lệnh curl để pushmetrics,
scrape metrics từ prometheus-sv khác.
